taxjar/taxjar-magento2-extension
# draft: chore: Correct PHPCS generic sniffs

### Context

Implement GitHub actions workflow.

### Description

Corrects PHPCS sniffs for: 
- `Generic.CodeAnalysis.UselessOverridingMethod.Found`
- `Generic.Files.LineLength.TooLong`
- `Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma`
- `Generic.Metrics.NestingLevel.TooHigh`
- `Generic.PHP.LowerCaseKeyword.Found`
- `Generic.WhiteSpace.DisallowTabIndent.TabsUsed`
- `Generic.WhiteSpace.ScopeIndent.Incorrect`

### Performance

N/A

### Testing

1. Search GH actions PHPCS output for `Generic.`
2. Observe no type `Generic` sniffs found

#### Versions
